.travis.yml

```diff
@@ -15,6 +15,9 @@ install:
   - python setup.py -q develop
   - pip install -r requirements-test.txt
 
+# command to run flake8
+before_script: "flake8 --ignore=E501,W291,W293 netdiff"
```

Comment:
This needs improvement, please proceed as follows:

1. set the line length to 110 characters in a similar fashion [this example setup.cfg file](https://github.com/django/django/blob/master/setup.cfg#L8) (it means adding a flake8 section and a `max-line-length = 110` line
2. remove the `--ignore` flag entirely
3. fix any issue that comes up